### DISCUSSION TITLE
translate-x not working on ios, but works on android

### DISCUSSION THREAD

**Discussion by Arinji2** [tailwindlabs/tailwindcss#11720] [Help]
Hello, so I made this site, and i added a sliding div. However this dosen't seem to work on iphone but works on android. Any Ideas?
Code: https://github.com/Arinji2/jardownloader

Working Version - On Android
https://github.com/tailwindlabs/tailwindcss/assets/87296817/8b1ad04e-1acd-4981-a9fd-21c5742fed64

Broken Version - On Iphone
https://github.com/tailwindlabs/tailwindcss/assets/87296817/9af73f3c-983c-476a-b492-3348ee5e9041

Any Idea?

**Comment by fedsdotlawl** [comment]
Same Issue.
